dist/library-conventions.yml

```diff
@@ -2,4 +2,4 @@ imports:
   - { resource: base-conventions.yml }
 
 parameters:
-  tasks.phpcs.standard: ./vendor/escapestudios/symfony2-coding-standard/Symfony2
+  tasks.phpcs.standard: ./vendor/squizlabs/php_codesniffer/src/Standards/PSR2
```

Comment:
This change is quite disruptive, I'd like to have a way to maintain backward compatibility, at least before rolling out 1.0.0, so that projects using this component will have the time to upgrade (there are quite a few of them).

Reply:
@ademarco Symfony2 standard is more strict than PSR-2. So, I think switching to PSR-2 will not break code written under Symfony2 standard.

Reply:
I also think it is more lenient now. At least in the project I was working on I am now no longer getting lots of false positives and my coding standards check is passing.

Which projects are using the PHP standard right now? It cannot be many since the current standard is very broken without this PR.

I'm in favor of going to a 1.0.0 release but there is at least one other ticket blocking that: https://webgate.ec.europa.eu/CITnet/jira/browse/OPENEUROPA-792

Reply:
Awesome, Packagist has the full list of packages that depend on this: https://packagist.org/packages/openeuropa/code-review/dependents

Reply:
The openeuropa/behat-transformation-context project also had this problem it seems, it worked creatively around the problem by extending the base conventions and specifying the PSR-2 standard in their `grumphp.yml.dist`: https://github.com/openeuropa/behat-transformation-context/blob/master/grumphp.yml.dist